_targets.R

```diff
@@ -5,27 +5,63 @@ source("3_visualize/src/plot_timeseries.R")
 
 options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
-
+#site numbers
+sites = c("01427207", "01432160","01435000", "01436690", "01466500")
+#parameter codes for query
+parameterCd = '00010' 
+startDate="2014-05-01" 
+endDate="2015-05-01"
+#parameters for plotting
 p_width <- 12
 p_height <- 7
 p_units <- "in"
 
 p1_targets_list <- list(
   tar_target(
-    site_data,
-    download_nwis_data(),
+    create_directories,
+    create_dirs(sites),
+    format = "file"
+  ),
+  tar_target(
+    site_data_1,
+    download_nwis_site_data(sites[1], create_directories,parameterCd, startDate, endDate),
+    format = "file"
+  ),
```

Comment:
when creating a target that creates a file the standard USGS practice is to name the target the file name with extension. So if you have site_data_1.csv you would put site_data_1_csv